.travis.yml

```diff
@@ -3,30 +3,49 @@ jdk: oraclejdk8
 env:
   global:
   - secure: fMBUkSsCBTw/U907w7Xm/yNzw4lq9yt7zFy0JyEZrA543LI7zQRIclIOtsDTUs0bwY+4KI08MNoYPLJ4TQhSJaH4wQ8b9C5TRsqXNfc0V718TcuYqSXqU6VPRVtX46/fCEWv/HAs0ksUKrHIlpWOwLPQpYDcKxwwlUndaakYKJ8=
+
 before_cache:
 - cd ${TRAVIS_BUILD_DIR}/gradle/caches/
 - find . -name "*.lock" -exec rm -rfv {} \;
 - cd ${TRAVIS_BUILD_DIR}
+
 cache:
   directories:
   - "${TRAVIS_BUILD_DIR}/gradle/caches/"
   - "${TRAVIS_BUILD_DIR}/gradle/wrapper/dists/"
+  
 notifications:
   email: false
+
 android:
   components:
   - tools
   - build-tools-25.0.2
   - platform-tools
   - tools
+  - android-22
+  - sys-img-armeabi-v7a-android-22
```

Comment:
What about adding some extras, which should probably enable voice recognition? However, this is not based on facts, just an idea :P I'm also not sure, which of these extras are already included :/ Also: We should first somehow make sure, that it's related to the play services :D

````
    - extra-google-google_play_services
    - extra-google-m2repository
    - extra-android-m2repository
    - extra-android-support
````

Reply:
Can we somehow check a build with these extras to see, if the voice recognition test is still skipped?